packages/trpc/server/routers/viewer/teams/roundRobin/getRoundRobinHostsToReasign.handler.ts

```diff
@@ -67,15 +68,26 @@ export const getRoundRobinHostsToReassign = async ({ ctx, input }: GetRoundRobin
       priority: host?.priority ?? 2,
     }));
 
-  const availableUsers = await ensureAvailableUsers(
-    { ...eventType, users: availableEventTypeUsers as IsFixedAwareUser[] },
-    {
-      dateFrom: dayjs(booking.startTime).format(),
-      dateTo: dayjs(booking.endTime).format(),
-      timeZone: eventType.timeZone || "UTC",
-    },
-    gettingRoundRobinHostsToReassignLogger
-  );
+  let availableUsers: IsFixedAwareUser[] = [];
+  try {
+    availableUsers = await ensureAvailableUsers(
+      { ...eventType, users: availableEventTypeUsers as IsFixedAwareUser[] },
+      {
+        dateFrom: dayjs(booking.startTime).format(),
+        dateTo: dayjs(booking.endTime).format(),
+        timeZone: eventType.timeZone || "UTC",
+      },
+      gettingRoundRobinHostsToReassignLogger
+    );
+  } catch (error) {
+    if (error instanceof Error && error.message === ErrorCode.NoAvailableUsersFound) {
```

Comment:
This isnt actually an error we care about here so we just set availableUsers to empty